core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/aggregate/AggregateOpDesc.scala

```diff
@@ -59,6 +59,8 @@ class AggregateOpDesc extends LogicalOp {
 
     val inputPort = InputPort(PortIdentity(0, internal = true))
 
+    val finalOutputPort = OutputPort(PortIdentity(0))
```

Comment:
is this blocking as well?

Reply:
Yes. But since we are naming materialization operators on the logical operator, keeping both local and global's outputs as blocking will be problematic. Let's keep it only for the global output port.